src/odemis/gui/cont/acquisition.py

```diff
@@ -2141,3 +2147,47 @@ def _on_ac_done(self, future):
         self._tab_panel.lbl_auto_center.Show()
         self._tab_panel.gauge_auto_center.Hide()
         self._sizer.Layout()
+
+
+class FastEMAlignmentController:
+    def __init__(self, tab_data, panel):
+        self._tab_data = tab_data
+        self._panel = panel
+
+        self._panel.align_gauge_progress.Hide()
+        panel.btn_align.Bind(wx.EVT_BUTTON, self._on_btn_align)
+
+        self._is_aligning = False
+
+    def _on_btn_align(self, evt):
+        if self._is_aligning:
+            logging.error("Cancelling alignment currently not supported.")
+            return
+        self._is_aligning = True
```

Comment:
Shouldn't more things be disabled in the GUI? I guess it wouldn't be a good idea to move the stage, or change the tab and start an overview acquisition during alignment. => Set the `is_acquiring` VA of the MainGUIData?